crates/redstone/src/helpers/mod.rs

```diff
@@ -1,2 +1,3 @@
 pub mod hex;
 pub mod iter_into;
+pub mod slice;
```

Comment:
I think helpers are intended for tests etc. Lets put this soewhere else. Hmm, maybe slice_ext mod?